### DISCUSSION TITLE
[1.3.0] OnRenderImage

### DISCUSSION THREAD

**Discussion by Deytooh** [naverz/zepeto-studio-global#16] [Q&A]
Hello, I already put that in the korean section but there is nothing more to say honestly xD 

This doesnt work on mobile, this work on Unity.
It will print "on render that!!" but Graphics.Blit will not work.
```
OnRenderImage(source: RenderTexture, destination: RenderTexture) { 
console.log("on render that!!"); 
Graphics.Blit(source, destination, this.material); 
}
```

This will work, the difference is that we didnt give any material to the Blit method, I guess the Blit method with material parameter is broken ?
```
OnRenderImage(source: RenderTexture, destination: RenderTexture) { 
console.log("on render that!!"); 
Graphics.Blit(source, destination); 
}
```

**Comment by jhslee** [comment]
Hello @Deytooh~ I am the US Tech Evangelist, John. Its nice to meet you! 👋 

I've tested the blit function on my end, and it seems to be working for me. 🤔 
The following is a test project I've created using a water ripple post-process shader to add a distortion ripple effect to the screen. 


![WaterRipplePostProcessing](https://user-images.githubusercontent.com/105688184/171551958-178bc1b8-efb3-45fc-b63a-5b77a2be4aa9.gif)
**[WaterRipple Post Processing Example]**

I wonder if rather than the function itself, it may be more of an issue with the shader. Perhaps the shader feature you're attempting to implement is not supported by our render pipeline.

I am attaching my Post Processing Example Project if you'd like to compare your settings with mine! If you have issues with the project file, please paste the `Assets > WaterRipplePostProcessing` folder into your working assets directory. 😄 

[WaterRipplePostProcessingExample.zip](https://github.com/naverz/zepeto-studio-global/files/8820911/WaterRipplePostProcessingExample.zip)

**Reply by Deytooh** [reply]
Hi John ! Nice to meet you :)
Thank you for you anwser !!
So, I checked with your project and indeed it works good. 
So I copied my shader in your shader to check ( why not it could be working haha )
And...it works ! Maybe something weird happened to my shader file haha

Anyway problem solved, thank you very much 💯 

Have a nice day !
